udata_front/theme/gouvfr/assets/js/pages/PublishingForm/PublishingForm.vue

```diff
@@ -108,7 +103,8 @@ const dataset = ref<NewDataset>({
     update_frequency: false,
     update_fulfilled_in_time: false,
   },
-  ...owned,
+  owner: null,
```

Comment:
Does this cause a type error ? owner and organization shouldn't be null at the same time